licensescout-core/src/main/java/org/aposin/licensescout/license/LicenseStoreData.java

```diff
@@ -198,26 +201,20 @@ private void addNoManualInformationLicense(final ILSLog log) {
      * @throws IOException
      */
     public void readUrlMappings(final InputStream inputStream, final ILSLog log) throws IOException {
-        String line = "";
-        final String cvsSplitBy = ",";
-
+        final CSVFormat csvFormat = CSVFormat.DEFAULT.withCommentMarker('#');
```

Comment:
Yes, CSVFormat.DEFAULT specifies ',' as the delimiter. Since this constant is something like a fixed configuration that follows an RFC, it is unlikely to change. However, to be on the safe side, I added a delimitr specification explicitly.